configs/prod/schemas/src/hbi.ksl

```diff
@@ -3,6 +3,9 @@ namespace hbi
 
 import rbac
 
+@rbac.add_v1_based_permission(app:'staleness', resource:'staleness', verb:'read', v2_perm:'staleness_read_staleness');
```

Comment:
I remembered after the call that the other permissions (and the patterns in the HBI proof of concept) are formatted '${application}_${resource}_${verb}' like the role permissions but use alternate verbs (read -> view, write -> update)

So we probably want staleness_staleness_view and staleness_staleness_update here.